opennlp-core/opennlp-runtime/src/test/java/opennlp/tools/sentdetect/SentenceDetectorMEGermanTest.java

```diff
@@ -45,19 +48,25 @@ public class SentenceDetectorMEGermanTest extends AbstractSentenceDetectorTest {
   
   private static SentenceModel sentdetectModel;
```

Comment:
Given `prepareResources(..)` is not static any longer, this field should also avoid "static" and become an instance per Test case. Please consider removing `static` here.